src/browser-frontend/views/browser/chrome/tabbar/tab.css

```diff
@@ -57,3 +57,14 @@ specific language governing permissions and limitations under the License.
   color: var(--theme-tab-active-color);
   transition: width 0.05s  0.15s var(--animation-curve), height 0.2s var(--animation-curve);
 }
+
+.tab.animate {
+  background-image: var(--theme-inactive-tab-background-loaded);
+  background-position: var(--theme-tab-inactive-background-position);
+  transition: none !important;
```

Comment:
it was used to prevent the sliding up background image from appearing above the flash gif when the tab becomes active/inactive. but it seems isn't needed any more.